### PR TITLE
Fixed incorrect link.

### PR DESCRIPTION
**Pull request by Doommius**
Fixed lint that points to GitLab instead of Azure pipelines.

Fixes #

## Proposed Changes

1. …
2. …
3. …

### Author/Contributor
- [x] Add entry to the [CHANGELOG](https://github.com/oxsecurity/megalinter/blob/main/CHANGELOG.md) listing the change and linking to the corresponding issue (if appropriate)
- [x] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

File: mega-linter-runner/generators/mega-linter/index.js [hunk 1]
```diff
@@ -275,7 +275,7 @@ When you don't know what option to select, please use default values`
       return;
     }
     this.log(
-      "Azure pipelines config generation not implemented yet, please follow manual instructions at https://megalinter.io/installation/#gitlab"
+      "Azure pipelines config generation not implemented yet, please follow manual instructions at https://megalinter.io/latest/installation/#azure-pipelines"
```
Comment (Doommius) [inline mega-linter-runner/generators/mega-linter/index.js:278]:
```suggestion
      "Azure pipelines config generation not implemented yet, please follow manual instructions at https://megalinter.io/installation/#azure-pipelines"
```

**Comment by echoix** [comment]
@nvuillam I'm not sure why the jsonschema needed to change. I thought that there might have been a documentation change with this PR, but not sure why the whole file was changed. Maybe remove the last commit?

**Comment by nvuillam** [comment]
Probably just formatting:)

**Comment by nvuillam** [comment]
CI is temporary broken, I don't know if it's because of my updates or because an upgraded linter that makes a mess ^^
Once solved we'll be able to merge this PR :)

**Review by nvuillam — APPROVED** [review]
Thanks fo the catch and for the PR :)

**Review by nvuillam — APPROVED** [review]
Thanks fo the catch and for the PR :)

**Review by nvuillam — APPROVED** [review]
Thanks fo the catch and for the PR :)

**Review by nvuillam — APPROVED** [review]
Thanks fo the catch and for the PR :)

**Pull request merged**